fuse_constraints/src/normal_prior_pose_2d.cpp

```diff
@@ -42,6 +42,7 @@ namespace fuse_constraints
 {
 
 NormalPriorPose2D::NormalPriorPose2D(const fuse_core::MatrixXd& A, const fuse_core::Vector3d& b) :
+  ceres::SizedCostFunction<ceres::DYNAMIC, 2, 1>();
```

Comment:
I'm going to start with the dumb question: Shouldn't this be a comma?
```suggestion
  ceres::SizedCostFunction<ceres::DYNAMIC, 2, 1>(),
```

Reply:
:facepalm: Of course. I didn't even build this locally, since I couldn't reproduce the error, and I didn't realized of that typo.

Fixed.